src/components/IssueListItem.js

```diff
@@ -8,6 +8,8 @@ import Octicon, {getIconByName} from "@primer/octicons-react";
 dayjs.extend(relativeTime);
 
 function IssueListItem({title, labels, author, opened, type, participants, comments, milestone}) {
+  const participantsToShow = 3;
+  const participantsShowDiff = participants && participants.totalCount - participantsToShow;
```

Comment:
Naming suggestion - **participantsToShowDelta**
Thoughts?

Reply:
@yg?

Reply:
I think it sounds too complex!

Reply:
Agreed

Reply:
I think `delta` is clever, but it may be confusing to use. 

I am fine with participantsToShowDiff or simply participantsDiffCount.

Reply:
I'm fine with both of these suggestions, up to you guys

Reply:
Nevermind. Delta is the mathematical term to tell the difference between two values.
As per dictionary.
![image](https://user-images.githubusercontent.com/1915009/85029417-e3e88300-b199-11ea-8d11-25b2182176c6.png)

Reply:
`participantsDiffCount` Sounds good!

Reply:
Let's go for it :)